include/gz/math/MecanumDriveOdometry.hh

```diff
@@ -134,18 +129,7 @@ namespace gz
       /// \return Right wheel radius in meters.
       public: double RightWheelRadius() const;
 
-
-#ifdef _WIN32
-// Disable warning C4251 which is triggered by
-// std::unique_ptr
-#pragma warning(push)
-#pragma warning(disable: 4251)
-#endif
-      /// \brief Private data pointer.
-      private: std::unique_ptr<MecanumDriveOdometryPrivate> dataPtr;
-#ifdef _WIN32
-#pragma warning(pop)
-#endif
+      GZ_UTILS_IMPL_PTR(dataPtr)
```

Comment:
```suggestion
      /// \brief Private data pointer.
      GZ_UTILS_IMPL_PTR(dataPtr)
```

Here and elsewhere

Reply:
https://github.com/gazebosim/gz-math/pull/608/commits/78a90662c12bcf00e340e3477184caeef656738a